README.md

```diff
@@ -1,4 +1,23 @@
 
+# Correlation Vector
+
+## Background
+
+**Correlation Vector** (a.k.a. **cV**) is a format and protocol standard for tracing and correlation of events through a distributed system based on a light weight vector clock.
+The standard is widely used internally at Microsoft for first party applications and services and supported across multiple logging libraries and platforms (Services, Clients - Native, Managed, Js, iOS, Android etc). The standard powers a variety of different data processing needs ranging from distributed tracing & debugging to system and business intelligence, in various business organizations.
+For more on correlation vector, read [future](https://github.com/Microsoft/CorrelationVector) roadmap and [scenarios](https://github.com/Microsoft/CorrelationVector/blob/master/Scenarios.md).
```

Comment:
For more on the correlation vector specification and the scenarios it supports, please see here